.eslintrc.json

```diff
@@ -9,7 +9,7 @@
     "ecmaVersion": 2020
   },
   "globals": {
-    "BigInt": true
+    "BigInt": "readonly"
```

Comment:
I think this is no longer needed in we use `"ecmaVersion": 2020` (see https://github.com/eslint/eslint/pull/11983).